src/translations.js

```diff
@@ -208,6 +208,25 @@ const translations = {
       speaker_management: '组合管理',
     },
   },
+  cancelAnimationFrame: {
```

Comment:
This should be the locale, mistaken autocomplete? 😉

Reply:
Ouch, sorry. Yes, it's a autocomplete mistaken. I've upadated the PR with problem solved.